internal/common/logs.go

```diff
@@ -153,3 +162,12 @@ func GenerateLogConfig(numberMonitoredLogs int, filePath string) error {
 
 	return nil
 }
+
+// getTempFolder gets the temp folder for generate logs
+// depends on the operating system
+func getTempFolder() string {
+	if runtime.GOOS == "windows" {
+		return "C:"
```

Comment:
nit: Windows should have an equivalent temp folder. I don't recommend we use the root drive.
```
C:\Users\Administrator\AppData\Local\Temp
```
or whatever is in `%TEMP%` env var might be better.

Reply:
Interesting !! Initially I thought we only need to write 1 or two files to this `C:\` folder so `C:\` is fine. I'm good with changing it to `C:\Users\Administrator\AppData\Local\Temp`